sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/ier12-factory.ts

```diff
@@ -122,6 +124,14 @@ export async function saveIER12TestInputData(
     testInputData.offering,
     studyStartDate,
     studyEndDate,
+    {
```

Comment:
I would prefer to have the `createFakeEducationProgramOffering` called from this method to have the `parentOffering` ready to be provided to `updateIER12OfferingFromTestInput`, if needed.
In this way, we would avoid the need to provide all the relations.